### PR TITLE
transition from deprecated `dplyr::*_at()` variants

### PR DESCRIPTION
**Pull request by simonpcouch**
🥑

### CODE REVIEW

**Comment by simonpcouch** [comment]
(Shamefully) tacking on [c642b5c](https://github.com/tidymodels/tune/pull/655/commits/c642b5cf1531b5d606837666753c40b3f4b8ba07) to this PR, as a separate PR would touch the same lines but would be a quick one. Use of an anonymous function drops into `dplyr:::as_inlined_function()`, where `quo()` machinery results in a pretty substantial slowdown.

File: R/collect.R [hunk 1]
```diff
@@ -269,6 +271,10 @@ prob_summarize <- function(x, p) {
   x
 }
 
+mean_na_rm <- function(x) {
```
Comment (hfrick) [inline R/collect.R:274]: Do you think it's worth it leaving a comment here that this is to avoid using an anonymous function for speed reasons?

**Review by hfrick — APPROVED** [review]
Good collection of commits 😜

File: R/collect.R (hunk 1 above)
Comment (simonpcouch) [reply R/collect.R:274]: Yes, good point.

Comment (simonpcouch) [reply R/collect.R:276]: [658a1dc](https://github.com/tidymodels/tune/pull/655/commits/658a1dc5679792419f73c86460de2ad8be44e3ae)!

**Pull request merged**